_framework/robust-definitions.markdown

```diff
@@ -0,0 +1,13 @@
+---
+component: Robust Definitions
+order: 2
+summary_version: "Robust and clear definitions of beneficial ownership should cover\
```

Comment:
We'll need to double check what happens when/if this runs through siteleaf's yaml parser, as it seems to do things differently for multiline strings, but I can do that before I publish it.